app/Http/Controllers/TransactionController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Inertia\Inertia;
+use Illuminate\Http\Request;
+use App\Models\transaction;
+use App\Http\Requests\TransactionRequest;
+use PDF;
+use Illuminate\Support\Facades\Redirect;
+
+class TransactionController extends Controller
+{
+    public function index(){
+        return Inertia::render('CreateTransaction');
+    }
+
+    public function show(){
+        return Inertia::render('TransactionList');
+    }
+
+    public function save(TransactionRequest $request){
+            $Transaction= new Transaction();
+            $Transaction->date = $request->date;
+            $Transaction->description = $request->description;
+            $Transaction->paid = $request->paid;
+            $Transaction->unit_amount = $request->unit_amount;
+            $Transaction->unit_quantity = $request->unit_quantity ;
+            $Transaction->unit_name = $request->unit_name;
+            $Transaction->type = $request->type;
+            $Transaction->status = $request->status;
+            $Transaction->utr = $request->utr;
+            $Transaction->comments = $request->comments;
+            $Transaction->project = $request->project;
+            $res= $Transaction-> save();
```

Comment:
```suggestion
            $res = $Transaction->save();
```

Reply:
Thank you for the suggestions. The changes shall reflect by latest commit.